plugins/record-hook/src/example.ts

```diff
@@ -0,0 +1,89 @@
+import { SheetConfig } from '@flatfile/api'
```

Comment:
```suggestion
import type { SheetConfig } from '@flatfile/api'
```

This is a minor but valuable point for plugins. By using `import type` the final compiled code does not import the library keeping it a lot smaller.